recipes/pfr/all/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  "1.0.4":
+    - url: "https://github.com/apolukhin/magic_get/archive/1.0.4.tar.gz"
+      sha256: "58fabdb95f8e8b9cc48a50569ff85a30227a8641c21b9a9fcf5cb0a36c1dd791"
+    - url: "https://www.boost.org/LICENSE_1_0.txt"

```

Comment:
I would go the patch way, so the recipe does not have to be modified **IF/WHEN** https://github.com/apolukhin/magic_get/pull/50 gets accepted.

Reply:
> 
> 
> I would go the patch way, so the recipe does not have to be modified **IF/WHEN** [apolukhin/magic_get#50](https://github.com/apolukhin/magic_get/pull/50) gets accepted.

Sounds totally reasonable, as we don't need these workaround.